python/hyperon/exts/snet_io/snet_gate.py

```diff
@@ -137,6 +141,10 @@ def get_service_messages(self):
     def get_operation_atom(self):
         return OperationAtom(self.service_details[1], self)
 
+    def open_channel_and_deposit(self, amount, expiration):
+        self.service_client.deposit_and_open_channel(amount, expiration)
```

Comment:
it would be nice to have an example of usage of this function

Reply:
So, should I add an example to the test_snet_call.metta?

Reply:
yes, but make it inactive (no `!`)

Reply:
Ok. Ive' added. Please check